client-js/app/scripts/registry.ts

```diff
@@ -26,165 +28,10 @@ export interface RegistryApi {
   moveDir(params: {path: Array<string>; dir: string; newPath: Array<string>; newDir: string}): Promise<RegistryListing>;
 }
 
-export class RegistryService implements RegistryApi {
-  baseUrl: string;
-
-  constructor(baseUrl: string) {
-    this.baseUrl = baseUrl + '/api/registry';
-  }
-
-  dir(params: {path: Array<string>}): Promise<RegistryListing> {
-    return this.sendRequest('/dir', params.path);
-  }
```

Comment:
There were two implementations before, one for the HTTP requests, one for the WebSocket, I just removed the HTTP ones